dashboard/test/factories/factories.rb

```diff
@@ -779,7 +779,7 @@
 
   factory :user_script do
     user {create :student}
-    script
+    script {create :script}
```

Comment:
nit: `association :script`

Reply:
What does this change do? (either adding `{create :script}` or `association :script`)

Reply:
It means the default factory value of `script` is a newly created script (from the script factory). 

In this case they do the same thing. The [association](https://github.com/thoughtbot/factory_bot/blob/master/GETTING_STARTED.md#associations) means FactoryBot will run that other factory if there's no provided value, and the [dynamic attribute](https://github.com/thoughtbot/factory_bot/blob/master/GETTING_STARTED.md#dynamic-attributes) block will be executed.

Reply:
I thought it was already implied. [The FactoryBot docs on associations say](https://www.rubydoc.info/gems/factory_bot/file/GETTING_STARTED.md#Associations)

> If the factory name is the same as the association name, the factory name can be left out.
>
> ```rb
> factory :post do
>  # ...
>  author
> end
> ```

...but it wouldn't be the first time I've looked at the wrong version of the docs.

Reply:
good catch, I've reverted the change to factory.rb since it appears to be unneeded.